packages/components/button/src/Button.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { StoryFn } from '@storybook/react'
+import { type ComponentProps } from 'react'
+
+import { Button } from '.'
+
+type ButtonProps = ComponentProps<typeof Button>
+
+export const Default: StoryFn = () => <Button>Default button</Button>
+
+const sizes: ButtonProps['size'][] = ['sm', 'md', 'lg']
+const intents: ButtonProps['intent'][] = [
+  'primary',
+  'secondary',
+  'success',
+  'alert',
+  'danger',
+  'neutral',
+  'surface',
+]
+const designs: ButtonProps['design'][] = ['filled', 'outlined', 'tinted', 'ghost', 'contrast']
+const shapes: ButtonProps['shape'][] = ['rounded', 'square', 'pill']
+
+export const Sizes: StoryFn = () => (
```

Comment:
I think that you need to handle `args` here or the code documentation won't work. See https://github.com/adevinta/spark/blob/main/packages/components/radio/src/Radio.stories.tsx#L6